css-ecommerce-page.tar/css-ecommerce-page/styles.css

```diff
@@ -0,0 +1,74 @@
+.container{
+  background-color: white;
+  height:200px;
+  display: block;
+}
+
+.row{
+  display: flex;
+  flex-wrap: wrap;
+}
+
+header{
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-weight: bolder;
+  font-size: large;
+  flex-wrap: wrap;
+}
+
+.column-fifth{
+  flex-basis: calc(100%/5);
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+}
+
+.row .column-fifth img{
+  height: 200px;
+  width: 100px;
+   display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  vertical-align: top;
+}
+
+.row .column-fifth h3{
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position:fixed;
+  bottom:20px;
+  font-size:large;
+  flex-wrap: wrap;
+  vertical-align: top;
+}
+
+.row .column-fifth p{
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: fixed;
```

Comment:
I think using `position: fixed` is messing with the layout and shoving your `h3` and `p` tags to the bottom of the page.

I actually don't think you really need additional styles on the `h3` and `p` tags at all since you want it to be in the document flow and part of the column that it is in.